docs/migrations/inplace.md

```diff
@@ -0,0 +1,532 @@
+2022-10-02 10:44:51,700 INFO    [Products.GenericSetup.tool:1469][waitress-3] Applying main profile profile-plone.staticresources.upgrades:213
+2022-10-02 10:44:51,701 INFO    [GenericSetup.rolemap:100][waitress-3] Role / permission map imported.
+2022-10-02 10:44:51,860 INFO    [Products.GenericSetup.tool:1207][waitress-3] Profile plone.staticresources:default upgraded to version '213'.
+2022-10-02 10:44:51,860 INFO    [plone.app.upgrade:300][waitress-3] Ran upgrade step: Upgrade profiles of core Plone modules to specific versions.
+2022-10-02 10:44:51,860 INFO    [Products.GenericSetup.tool:1428][waitress-3] Importing profile profile-plone.app.upgrade.v60:to6007 with dependency strategy ignore.
+2022-10-02 10:44:51,860 INFO    [Products.GenericSetup.tool:1469][waitress-3] Applying main profile profile-plone.app.upgrade.v60:to6007
+2022-10-02 10:44:51,861 INFO    [GenericSetup.rolemap:100][waitress-3] Role / permission map imported.
+2022-10-02 10:44:51,865 INFO    [GenericSetup.actions:100][waitress-3] Actions tool imported.
+2022-10-02 10:44:51,867 INFO    [plone.app.upgrade:300][waitress-3] Ran upgrade step: Run to6007 upgrade profile.
+2022-10-02 10:44:51,867 INFO    [plone.app.upgrade:300][waitress-3] Ran upgrade step: Add a timezone property to portal memberdata if it is missing.
+2022-10-02 10:44:51,867 INFO    [plone.app.upgrade:300][waitress-3] Ran upgrade step: Fix the portal action icons.
+2022-10-02 10:44:51,867 INFO    [plone.app.upgrade:300][waitress-3] Ran upgrade step: Rename the behavior collective.dexteritytextindexer to plone.textindexer
+2022-10-02 10:44:51,867 INFO    [plone.app.upgrade:313][waitress-3] End of upgrade path, main migration has finished.
+2022-10-02 10:44:51,867 INFO    [plone.app.upgrade:319][waitress-3] Starting upgrade of core addons.
+2022-10-02 10:44:51,869 INFO    [plone.app.upgrade:321][waitress-3] Done upgrading core addons.
+2022-10-02 10:44:51,869 INFO    [plone.app.upgrade:357][waitress-3] Your Plone instance is now up-to-date.
+```
+````
+
+After that step you should check if you have add-ons that need to be upgraded in the add-ons control panel `/@@prefs_install_products_form`.
+
+In this example `collective.easyform` can be upgraded (from version 3.2.0 to 4.1.0):
+
+```{image} _static/prefs_install_products_form.png
+:alt: The view /@@prefs_install_products_form showing that easyform needs to be upgraded
+```
+
+After clicking on {guilabel}`Upgrade collective.easyform`, the add-on is up-to-date.
+The log of your Plone instance will show what was done during that upgrade:
+
+```console
+2022-10-02 10:58:12,920 INFO    [collective.easyform.upgrades:92][waitress-1] Removed easyform.migrate_all_forms registry record
+2022-10-02 10:58:12,921 INFO    [Products.GenericSetup.tool:1207][waitress-1] Profile collective.easyform:default upgraded to version '1013'.
+```
+
```

Comment:
The migration steps for add'ons are nit in plone.app.upgrade but are contained in the individual python package of the add'on and maintained by its developers.